src/providers/BackgroundBlurProvider/index.tsx

```diff
@@ -158,12 +158,10 @@ export const BackgroundBlurProvider: FC<React.PropsWithChildren<Props>> = ({
     );
     const currentProcessor = await initializeBackgroundBlur();
     try {
-      const logger =
-        options?.logger ||
-        new ConsoleLogger('BackgroundBlurProvider', LogLevel.INFO);
+      const optionsLogger = options?.logger || logger;
```

Comment:
This is after the initialize right? We pass the logger to DefaultVideoTransformDevice